dev-tools/mage/gotest.go

```diff
@@ -149,7 +177,7 @@ func DefaultGoTestIntegrationFromHostArgs() GoTestArgs {
 // GoTestIntegrationArgsForModule returns a default set of arguments for running
 // module integration tests. We tag integration test files with 'integration'.
 func GoTestIntegrationArgsForModule(module string) GoTestArgs {
```

Comment:
This should probably be `GoTestIntegrationArgsForPackage` with a `pkg` arg instead